src/isPalindrome.js

```diff
@@ -20,6 +20,8 @@
  */
 function isPalindrome(str) {
   // write code here
+  const onlyLett = str.toLowerCase().replace(/[^\w]/g, '');
```

Comment:
'Lett' is not a typical abbreviation. Consider renaming this to alphabeticChars, or alphabeticLetters